### PR TITLE
Replace all logs in api module to use zerolog

### PR DESCRIPTION
**Pull request by hsienchiaolee**
## Issue

#1171 

## Test
N/A, ran `./test/test_before_submit.sh`

## TODO
- Merge #1191 and rebase `master`

### CODE REVIEW

**Comment by harmony-ek** [comment]
Jenkins speaking; can one of the admins verify this patch?

**Comment by mikedoan** [comment]
Good work, Kai.

**Comment by LeoHChen** [comment]
Jenkins, test this please.

File: api/proto/message/server.go [hunk 1]
```diff
@@ -60,7 +59,7 @@ func (s *Server) Process(ctx context.Context, message *Message) (*Response, erro
 		for _, balance := range balances {
 			stringBalances = append(stringBalances, balance.String())
 		}
-		utils.GetLogInstance().Info("getPlayers", "players", players, "balances", stringBalances)
+		utils.Logger().Info().Strs("players", players).Strs("balances", stringBalances).Msg("getPlayers")
```
[the remark this answers is not in the document]
Comment (hsienchiaolee) [reply api/proto/message/server.go]: Yes, default logger comes with file/line (not function/line). Calling `.Msg()` is required on ZeroLog for it to trigger the actual output. We can replace with more meaningful message instead.

**Comment by mikedoan** [comment]
Jenkins, test this please.

**Comment by LeoHChen** [comment]
Jenkins, test this please.

File: api/proto/message/server.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (mikedoan) [reply api/proto/message/server.go]: I'll fix leo feedback later.

File: api/proto/message/server.go [hunk 2]
```diff
@@ -45,11 +44,11 @@ func (s *Server) Process(ctx context.Context, message *Message) (*Response, erro
 
 		key, err := crypto.HexToECDSA(priKey)
 		if err != nil {
-			utils.GetLogInstance().Error("Error when HexToECDSA")
+			utils.Logger().Error().Msg("Error when HexToECDSA")
 		}
 		address := crypto.PubkeyToAddress(key.PublicKey)
 
-		utils.GetLogInstance().Info("Enter:", "amount", amount, "for address", address)
+		utils.Logger().Info().Int64("amount", amount).Bytes("address", address[:]).Msg("Enter")
```
[the remark this answers is not in the document]
Comment (mikedoan) [reply api/proto/message/server.go]: I'll fix leo feedback later.

**Pull request merged**